KubeArmor/core/kubeArmor.go

```diff
@@ -336,11 +336,12 @@ func KubeArmor() {
 		dm.Node.Annotations = map[string]string{}
 		dm.HandleNodeAnnotations(&dm.Node)
 
+		dm.Node.OSImage = kl.GetCommandOutputWithoutErr("lsb_release", []string{"-d", "-s"})
```

Comment:
`lsb_release` is not available on most platforms. Mostly debian/ubuntu it is available.
A more portable way to check this is using `hostnamectl`.

Reply:
used hostnamectl instead of lsb_release